src/test/integration/controllers/accounts.controller.e2e-spec.ts

```diff
@@ -0,0 +1,252 @@
+
+                    await request(app.getHttpServer())
+                        .get(`${path}?${filter}=${value}&${order}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(res.body).toBeDefined();
+                            expect(res.body[0].timestamp).toBeGreaterThanOrEqual(res.body[1].timestamp);
+                        });
+                });
+            });
+        });
+
+        test.each`
+        size
+        ${1}
+        ${55}
+        ${4}
+        ${10000}`
+            (
+                `should return a list of $size items`,
+                async ({ size }) => {
+                    const params = new URLSearchParams({
+                        'size': size,
+                    });
+
+                    await request(app.getHttpServer())
+                        .get(`${path}?${params}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(res.body).toHaveLength(size);
+                        });
+                }
+            );
+    });
+
+    describe('/accounts/count', () => {
+        it('should return count of all blocks from all shards', async () => {
+            await request(app.getHttpServer())
+                .get(`${path}/count`)
+                .expect(200)
+                .then(res => {
+                    expect(+res.text).toBeGreaterThanOrEqual(2461134);
+                });
+        });
+
+        [
+            {
+                ownerAddress: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqplllst77y4l',
+                count: 1,
+            },
+            {
+                ownerAddress: 'erd1cc2yw3reulhshp3x73q2wye0pq8f4a3xz3pt7xj79phv9wm978ssu99pvt',
+                count: 22,
+            },
+            {
+                ownerAddress: 'erd1x45vnu7shhecfz0v03qqfmy8srndch50cdx7m763p743tzlwah0sgzewlm',
+                count: 4,
+            },
+            {
+                ownerAddress: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqylllslmq6y6',
+                count: 57,
+            },
+        ].forEach(({ ownerAddress, count }) => {
+            describe(`ownerAddress = ${ownerAddress}`, () => {
+                it(`should return count of accounts searched by owner address ${ownerAddress}`, async () => {
+                    const params = new URLSearchParams({
+                        'ownerAddress': `${ownerAddress}`,
+                    });
+
+                    await request(app.getHttpServer())
+                        .get(`${path}/count?${params}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(+res.text).toBeGreaterThanOrEqual(count);
+                        });
+                });
+            });
+        });
+    });
+
+    describe('/accounts/{address}', () => {
+        it('should return account details for a given address', async () => {
```

Comment:
Please retake this test case because, sometimes will throw fail
Especially for the res.body.balance, because this balance can change over time